### ISSUE TITLE
How to deal with Popover_LowBattery_Base_Header?

### ISSUE THREAD

**Issue by TheDeveloperGuy** [ValveSoftware/openvr#630]
Developing a driver for an HMD with a battery powered base station. Put all of the battery reporting info into the driver, but when the base station battery is low, I get Popover_LowBattery_Base_Header and Popover_LowBattery_Base_Description when I hover over the icon instead of an actual message. 

Something for me to fix/implement or something for Valve to fix/implement? IIRC from the last time I had one of these "Popover_" type messages it is a Valve issue.

**Comment by HelenXR** [comment]
Would you like to change the icon when the battery is low?

**Comment by TheDeveloperGuy** [comment]
No, I'd like SteamVR to not give these kind of messages. So either I'm reporting a bug, or asking for a method to resolve this.
